appjson/healthcheck.go

```diff
@@ -25,16 +27,23 @@ type AppJSON struct {
 }
 
 type Healthcheck struct {
-	Attempts     int      `json:"attempts,omitempty"`
-	Command      []string `json:"command,omitempty"`
-	Content      string   `json:"content,omitempty"`
-	InitialDelay int      `json:"initialDelay,omitempty"`
-	Name         string   `json:"name,omitempty"`
-	Path         string   `json:"path,omitempty"`
-	Timeout      int      `json:"timeout,omitempty"`
-	Type         string   `json:"type,omitempty"`
-	Uptime       int      `json:"uptime,omitempty"`
-	Wait         int      `json:"wait,omitempty"`
+	Attempts     int        `json:"attempts,omitempty"`
+	Command      []string   `json:"command,omitempty"`
+	Content      string     `json:"content,omitempty"`
+	InitialDelay int        `json:"initialDelay,omitempty"`
+	Name         string     `json:"name,omitempty"`
+	Path         string     `json:"path,omitempty"`
+	Timeout      int        `json:"timeout,omitempty"`
+	Type         string     `json:"type,omitempty"`
+	Uptime       int        `json:"uptime,omitempty"`
+	Wait         int        `json:"wait,omitempty"`
+	OnFailure    *OnFailure `json:"onFailure,omitempty"`
+}
+
+type OnFailure struct {
+	Command       []string `json:"command,omitempty"`
+	Url           string   `json:"url,omitempty"`
+	OnNonzeroExit bool     `json:"onNonzeroExit,omitempty"`
```

Comment:
We can probably just remove this. Idk what I was thinking in the initial spec, but this is unused in your diff.